src/components/button/_button.scss

```diff
@@ -204,10 +221,15 @@
       }
     }
 
-    &:hover,
-    &:focus {
+    &:hover {
       background-color: $govuk-warning-button-hover-colour;
     }
+
+    &:focus {
+      color: $govuk-focus-text-colour;
+      background-color: $govuk-focus-colour;
+      box-shadow: 0 2px 0 $govuk-focus-text-colour;
```

Comment:
Similarly to above comment these are also inherited from `.govuk-button`

Reply:
The reason there are duplicates is because the secondary buttons :hover state overrides `.govuk-button` focus because its further down the page.

Reply:
Thanks Al. It seems that they set the same styles so I don't think they need to be overridden?

<img width="387" alt="Screen Shot 2019-05-10 at 15 16 41" src="https://user-images.githubusercontent.com/5007934/57533920-c01c0200-7336-11e9-925e-a0cdf5a74efe.png">

Reply:
Thanks for the IRL @aliuk2012 🙌 -  so, if we don't set the focus states here, the hover state that's being set for secondary buttons overrides it if button is being hovered while focused.

Probably worth leaving a comment, maybe something like
```
Set .govuk-button :focus styles to avoid :hover styles overriding them if button is both focused and hovered.
```

We could also consider moving the `.govuk-button` focus styles to the bottom of `button.scss` to avoid having to rewrite styles but let's think about that later.

Reply:
I think we have a better solution now that we have a separate `:focus:hover` state. I have since removed this code in the last commit.